stdlib/sys.mli

```diff
@@ -94,6 +94,18 @@ external time : unit -> (float [@unboxed]) =
 external chdir : string -> unit = "caml_sys_chdir"
 (** Change the current working directory of the process. *)
 
+external mkdir : string -> int -> unit = "caml_sys_mkdir"
+(** Create a directory with the given permissions.
```

Comment:
Do we add a comment saying that the permissions are ignored on Windows?

Reply:
We don't in `Unix.mkdir`, but that's not necessarily a reason not to!

Reply:
Similarly we don't for `open_in_gen` and friends

Reply:
OK, let's continue with the tradition :)

Reply:
I believe `open_in_gen` and friends will create readonly files under Windows if the permissions argument says so, hence it is not ignored.

Reply:
By ignored, I mean that the differences between Windows and Unix behaviour are ignored in the documentation - for example, I haven't verified that behaviour matches docs, but specifying `0o444` should raise an exception if you compile with the (Microsoft) debug runtime, while still creating a readonly file with a normal runtime (`0o4` should work without error on both), similarly with `0o555` or `0o5`